app/genesis.go

```diff
@@ -8,6 +8,7 @@ import (
 	cmn "github.com/tendermint/tmlibs/common"
 )
 
+// LoadGenesis - Load the genesis json file
```

Comment:
I'd prefer `LoadGenesis loads the genesis file into memory` (the same style is used in stdlib)